.github/workflows/ci.yml

```
@@ -211,17 +211,15 @@ jobs:
         apt-get -y update
         apt-get -y upgrade
 
-
-
     - name: Configure [Docker]
       run: |
         mkdir -p build
         cd build
-        cmake -C ${GITHUB_WORKSPACE}/.ci/initial-cache.gh.cmake  -G"${{ matrix.cmake_generator }}" -DNON_INTERACTIVE_BUILD:BOOL=TRUE -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}  ${{ matrix.project_tags_cmake_options }}  ..
+        cmake -DROBOTOLOGY_USES_GAZEBO:BOOL=OFF -C ${GITHUB_WORKSPACE}/.ci/initial-cache.gh.cmake  -G"${{ matrix.cmake_generator }}"  -DROBOTOLOGY_USES_GAZEBO:BOOL=OFF -DNON_INTERACTIVE_BUILD:BOOL=TRUE -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}  ${{ matrix.project_tags_cmake_options }}  ..
```

```suggestion
        cmake  -C ${GITHUB_WORKSPACE}/.ci/initial-cache.gh.cmake  -G"${{ matrix.cmake_generator }}" -DROBOTOLOGY_USES_GAZEBO:BOOL=OFF -DNON_INTERACTIVE_BUILD:BOOL=TRUE -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}  ${{ matrix.project_tags_cmake_options }}  ..
```

Is there a reason why `-DROBOTOLOGY_USES_GAZEBO:BOOL=OFF` is repeated?

---

I tried to set it both before and after and the `-C ${GITHUB_WORKSPACE}/.ci/initial-cache.gh.cmake ` parameter as the initial cache there instead sets ROBOTOLOGY_USES_GAZEBO to `ON`, but apparently this is not possible. Let's find another solution.